### DISCUSSION TITLE
Patch kitty for old OpenGL

### DISCUSSION THREAD

**Discussion by sarmong** [kovidgoyal/kitty#5774] [General]
I have an old thinkpad x201 and it has OpenGL 2.0. I know that kitty doesn't support it.

However, I don't really want to set up another terminal emulator, so my question is:

How can I patch kitty to be used on older OpenGL versions (obviously at my own risk) and what features will stop working?

**Comment by kovidgoyal** [comment]
No idea, have never even looked at supporting more than decade old versions of OpenGL. kitty uses a few functions introduced in 3.3 as wella s some shader features. Am fairly sure it would be a significant amount of work to try to get it running on OpenGL 2.0
